assets/locales/active.cy.toml

```diff
@@ -420,13 +420,13 @@ description = "Accept all cookies"
 one = "Accept all cookies"
 
 [CookiesBannerSetPreferencesAction]
-description = "Set Cookie Preferences"
-one = "Set Cookie Preferences"
+description = "Set cookie preferences"
+one = "Set cookie preferences"
```

Comment:
Think we need to be careful with the use of singular sentences. 

Example here, This is a plural sentence rather than singular, singular would be:
"Set cookie preference" ( which we don't need - at least yet) worth putting this in one of the plural options instead and using that. Otherwise, if we do need the singular version of this we would have to change all places this is used in addition to adding the singular.